lib/pbench/cli/server/shell.py

```diff
@@ -1,14 +1,34 @@
 #!/bin/env python3
 
-from pbench.server.api import create_app
 import subprocess
+import sys
+
+from configparser import NoSectionError, NoOptionError
+
+from pbench.common.exceptions import BadConfig, ConfigFileNotSpecified
+from pbench.server.api import create_app, get_server_config
+
+
+try:
+    server_config = get_server_config()
```

Comment:
Do they really need to be the same server config object?

Why not have:

```
def app():
    return create_app(get_server_config())

def main():
    server_config = get_server_config()
    try:
        ...
```

Reply:
In that case we would be creating two server config object instances where one instance is used just for getting gunicorn host port and workers information, is it fine?

Reply:
Personally, I'd rather that than the global variable.  That global variable is created a module load time, which is not ideal and creates an inflexible interface.